doc/source/guide/merging.rst

```diff
@@ -42,6 +42,13 @@ With the :option:`--merge-mode-functions` you can change this:
 - ``merge-use-line-max``: Allow same function on different lines, in this case use maximum line.
 - ``separate``: Allow same function on different lines. Instead of merging keep the functions separate.
 
+If you have the mangled or demangled name defined only in some files (mainly if you user cross compiling
```

Comment:
If you have the mangled or demangled name only defined in some files (if you mix reports generated using ``gcc-14`` and older ``gcc`` versions) the mangled names could only be present in one input file but not the other. When combining the reports, the default behavior is to detect the situation where the names do no match.